### ISSUE TITLE
Connection refused for URI https://api.amazon.com/auth/o2/token

### ISSUE THREAD

**Issue by SARANGHAJA** [jlevers/selling-partner-api#177]
Dear Sir,

I was developing on an old Windows PC, and due to circumstances, I copied all the source codes to another Windows PC and then ran ReportsApi->createReport(), and the following error appeared.

`Exception when calling ReportsApi->createReport: Connection refused for URI https://api.amazon.com/auth/o2/token`

Composer is not installed on the Windows PC in question.
(I did not install it because I have already copied all the source codes.)

I can't quite figure out why.

Please advise.
thank you.

**Comment by jlevers** [comment]
I'm not sure what would cause that, but I recommend installing Composer on the new computer, deleting the `vendor/` folder, and running `composer install`. There are probably various system-specific linked files that get messed up when you copy the files.

**Comment by SARANGHAJA** [comment]
Thank you for your answer.
It was confirmed that a vendor was created by installing the composer and executing "composer install" in the downloaded library folder.
However, the same error appears when executing.
`Exception when calling ReportsApi->createReport: Connection refused for URI https://api.amazon.com/auth/o2/token`

So...

After deleting the library, and downloaded a new library, deleted the composer.json file in the folder, and ran "composer request jlevers/selling-partner-api" to confirm vendor creation.
But the result is the same error.

I don't know what the problem is.
I'm going to work on the Windows PC and move to the Linux system soon... I'm worried.

**Comment by SARANGHAJA** [comment]
The problem was solved after reinstalling the FTP FileZil** client and remote TeamVie**, which had been installed due to other problems with the PC in question.

So I deleted the installed Composer and all source codes, copied all source codes from my old PC, and ran them as they were, and they ran normally.


I don't know what the problem was.
However, there is no other than the re-installation of the above program.
I'm glad it went well, but I'm worried that it will be another issue when I move to the Linux system later.

I hope it goes well.
